src/vision/roadmap/polish.md

```diff
@@ -2,7 +2,37 @@
 
 ## Impact
 
+* Users can predict and understand why the compiler raises error messages. Errors are aligned with an experienced user's intuition about how Rust works.
+* Error messages identify common misconceptions, suggest solutions, and are generally on par with sync Rust.
+  * Errors not only show that there is a problem, they help the user to fix it and to learn more about Rust (possibly directing the user to other documentation).
+  * The compiler may suggest crates from the ecosystem to help solve problems when appropriate.
 * Lints guide the user away from common errors and help them both to get started with async Rust and to maintain async Rust programs over time.
-* Errors not only show that there is a problem, they help the user to fix it and to learn more about Rust (possibly directing the user to other documentation).
-* The generated code from the compiler is high quality and performant.
-* Integration with low-level tooling and the like is high-quality.
\ No newline at end of file
+* Rust's async implementation is high quality and reflects an attention to detail.
+  * No internal compiler errors
+  * Compiler analysis and code generation passes are precise and not unnecessarily conservative.
+  * Integration with low-level tooling and the like is high-quality.
+  * The generated code from the compiler is high quality and performant.
+
+## Milestones
+
+| Milestone                                  | State | Key participants |
+| ---                                        | ---   | ---              |
+| Precise generator captures                 | 🦀    | [eholk]          |
+| ↳ Prototype                                | 🦀    | [eholk]          |
+| ↳ Documented in Rust Reference             | 🦀    | [eholk]          |
+| ↳ [Lang team] signoff                      | 💤    | [Lang team]      |
+| ↳ Stabilized                               | 💤    | [eholk]          |
+| Lint: [Large copies], large generators     | 💤    | |
+| ↳ [Lang team] initiative proposal          | 💤    | |
+| ↳ Implementation                           | 💤    | |
+| Lint: [Must not suspend]                   | 💤    | |
+| ↳ Landed the [RFC]                         | 💤    | |
```

Comment:
Needs implementation?

Reply:
I'll rephrase as "Implemented the RFC." The RFC is already merged, so I think all we need to do is implement what the RFC says.